nmap.go

```diff
@@ -1556,3 +1556,19 @@ func WithUnprivileged() Option {
 		s.args = append(s.args, "--unprivileged")
 	}
 }
+
+// WithNmapOutput makes nmap output standard output to the filename specified.
```

Comment:
This function will have no effect when using the `RunWithStreamer` function. I would suggest adding one more comment to the description.

Reply:
@elivlo What are you seeing that isn't working? I executed basic_scan_streamer_interface and the functionality appears to be working normally.

Reply:
Sure that was my bad. It is still working nomally.